{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"provenance":[],"authorship_tag":"ABX9TyMIeRk4n1iuUrKH0C9+b6Zk"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"markdown","source":["1. Escribe un programa que solicite al usuario dos números e intente dividir el primero entre el segundo. Maneja la excepción si el segundo número es cero y muestra un mensaje adecuado."],"metadata":{"id":"K5HIDhvAWgoi"}},{"cell_type":"code","execution_count":null,"metadata":{"id":"N5Bugc0lWWe4"},"outputs":[],"source":[]},{"cell_type":"markdown","source":["2. Escribe un programa que solicite al usuario que introduzca un número entero. Si el usuario introduce algo que no es un número entero, lanza una excepción y muestra un mensaje adecuado."],"metadata":{"id":"4mIipiavWiby"}},{"cell_type":"markdown","source":["3. Modifica el programa anterior para que maneje las excepciones ValueError y ZeroDivisionError. Si el usuario introduce un número no válido, muestra un mensaje, y si intenta dividir por cero, muestra otro mensaje."],"metadata":{"id":"4JAPbYF7Wivr"}},{"cell_type":"markdown","source":["4. Escribe un programa que intente abrir un archivo, lea su contenido y lo muestre. Utiliza finally para asegurarte de que el archivo siempre se cierre, independientemente de si ocurre una excepción."],"metadata":{"id":"yynSomajWi_7"}},{"cell_type":"markdown","source":["5. Crea una excepción personalizada llamada EdadInvalidaError. Luego, escribe un programa que solicite al usuario que introduzca su edad. Si la edad es negativa o mayor de 120, lanza la excepción personalizada con un mensaje de error adecuado.\n"],"metadata":{"id":"0J0Q_B5mWZ-a"}}]}